beluga_tools/setup.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages, setup
+
+package_name = 'beluga_tools'
+
+setup(
+    name=package_name,
+    version='0.0.0',
```

Comment:
@serraramiro1 consider matching this version with that of the `package.xml`

Reply:
Done. Changed `package.xml` one to be 0.0.0 too.